include/eve/traits/overload/default_behaviors.hpp

```diff
@@ -20,7 +21,29 @@ namespace eve
   }
 
   //====================================================================================================================
-
+  //! @addtogroup extensions
+  //! @{
+  //!   @struct callable
+  //!   @brief CRTP base class defining an EVE's @callable.
+  //!
+  //!   **Defined in Header**
+  //!
+  //!   @code
+  //!   #include <eve/traits/overload.hpp>
+  //!   @endcode
+  //!
+  //!   All EVE @callable use a similar protocol to find and call proper architecture or type specific implementation.
+  //!   This protocol includes options checks and management and proper level of error reporting.
+  //!   eve::callable is the most general base class for defining such a @callable without having to manually handles
+  //!   all these details.
+  //!
+  //!   @tparam Func          Type of current @callable being implemented.
+  //!   @tparam OptionsValues Type of stored options.
+  //!   @tparam Options       List of supported option specifications.
+  //!
+  //!   @groupheader{Example}
+  //!   @godbolt{doc/traits/callable_object.cpp}
```

Comment:
i think this file is already there, right?

Reply:
I just added the documentation for it. I don't get your remark.

Reply:
OK, the file is already there